lib/view/item_widget.dart

```diff
@@ -19,11 +20,40 @@ class _ItemWidgetState extends State<ItemWidget> {
   Widget build(BuildContext context) {
     final itemTextStyle = Theme.of(context).textTheme.titleLarge;
     return ListTile(
+      leading: IconButton(
```

Comment:
Don't put the delete button at the beginning of the item. It looks very strange. At the end of the item is better, but at the same time it doesn't look great to have so many delete icons on the screen.  
Do you have the Gmail or Google Messages app on Android? Experiment with them and see how they implement delete functionality. It is usually through some kind of press and hold action.  I think this would be complicated to implement, so we need something simpler.

Some ideas:
* Use the [`flutter_slidable`](https://pub.dev/packages/flutter_slidable) library to implement a slide gesture that displays the delete button.
* Use a modal bottom sheet ([docs here](https://api.flutter.dev/flutter/material/showModalBottomSheet.html)) to display the delete button. When the user presses and holds on an item, display the bottom sheet for the item. Currently the bottom sheet would only contain the delete button. In the future, you could add more buttons to the bottom sheet.

Reply:
I decided to go with your idea of a long press bringing up a modal bottom sheet.